common/djangoapps/xblock_django/user_service.py

```diff
@@ -15,6 +17,7 @@ class DjangoXBlockUserService(UserService):
     def __init__(self, django_user, **kwargs):
         super(DjangoXBlockUserService, self).__init__(**kwargs)
         self._django_user = django_user
+        self._user_is_staff = kwargs.get('user_is_staff', False)
```

Comment:
Can you check out the discussion here: https://groups.google.com/forum/#edx-code/agwShOD05mQ/cd5qN5SiikAJ

Rather than passing in `_user_is_staff`, it might be better to actually expose that as an attribute on the user returned by `get_current_user`.